src/routes/examples/add-remove-item/+page.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+	import Grid, { GridItem, gridHelper } from '$lib';
+
+	let items = [
+		{ id: crypto.randomUUID(), x: 0, y: 0, w: 2, h: 5 },
+		{ id: crypto.randomUUID(), x: 2, y: 2, w: 2, h: 2 },
+		{ id: crypto.randomUUID(), x: 2, y: 0, w: 1, h: 2 },
+		{ id: crypto.randomUUID(), x: 3, y: 0, w: 2, h: 2 },
+		{ id: crypto.randomUUID(), x: 4, y: 2, w: 1, h: 3 },
+		{ id: crypto.randomUUID(), x: 8, y: 0, w: 2, h: 8 },
+		{ id: crypto.randomUUID(), x: 4, y: 5, w: 1, h: 1 },
+		{ id: crypto.randomUUID(), x: 2, y: 6, w: 3, h: 2 },
+		{ id: crypto.randomUUID(), x: 2, y: 4, w: 2, h: 2 }
```

Comment:
GridItems inits its own id with `crypto.randomUUID()`, so it can be removed here. If you need it, you can use svelte's `let:id` mechanism

Reply:
I explicitly declared `id: crypto.randomUUID()` for demonstration purpose. But you are right, maybe it's not really needed.

Reply:
actually, i think is necessary when doing `#each items as item (item.id)` to keep a reference from outside the Grid Component. Maybe i need a little bit of help here because `let:id` is giving me undefined

Reply:
> actually, i think is necessary when doing `#each items as item (item.id)` to keep a reference from outside the Grid Component. Maybe i need a little bit of help here because `let:id` is giving me undefined

My bad. I forgot to pass id, active etc to <slot />. Ill open an issue here

Reply:
yes, but still it will only be available at GridItem level, and the id is needed before. at `#each items as item (item.id)`. If id is not passed then after removing or adding items will cause unwanted behaviour :(

Reply:
Good point. Cant argue with that.